templates/instance.interface.hbs

```diff
@@ -12,25 +12,46 @@
         </div>
         <div id="collapse{{@index}}" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading{{@index}}" data-index="{{@index}}">
           <div class="panel-body">
-            {{#each variables}}
+            {{#each variables as |variable|}}
             <div class="form-group">
               <div class="col-sm-4 control-label">
                 <label>{{ description }}</label>
               </div>
               <div class="col-sm-8">
                 {{#if_eq type "color"}}
-                <div class="input-group" colorpicker-component>
-                  <div class="input-group-addon"><i style="background-color:{{ setValue }}"></i></div>
-                  <input class="form-control" type="text" data-type="{{ type }}" name="{{ name }}" placeholder="{{ default }}" value="{{ setValue }}">
-                  {{#if helper}}
-                  <p class="text-helper">{{ helper }}</p>
-                  {{/if}}
-                </div>
+                  <div class="input-group" colorpicker-component>
+                    <div class="input-group-addon"><i style="background-color:{{ setValue }}"></i></div>
+                    <input class="form-control" type="text" data-type="{{ type }}" name="{{ name }}" placeholder="{{ default }}" value="{{ setValue }}">
+                    {{#if helper}}
+                    <p class="text-helper">{{ helper }}</p>
+                    {{/if}}
+                  </div>
                 {{else}}
-                <input class="form-control" type="text" data-type="{{ type }}" name="{{ name }}" placeholder="{{ default }}" value="{{ setValue }}" />
-                {{#if hint}}
-                <p class="text-helper">{{ hint }}</p>
-                {{/if}}
+                  {{#if_eq type "font"}}
+                    <label for="drop-down-{{ name }}" class="select-proxy-display">
+                      <span class="icon fa fa-chevron-down"></span>
+                      <span class="select-value-proxy">-- Select a font</span>
+                      <select id="drop-down-{{ name }}" data-type="{{ type }}" name="{{ name }}" class="hidden-select form-control">
+                        {{#if ../../customFonts}}
+                          {{#each ../../customFonts as |font|}}
```

Comment:
@squallstar How are is the `customFonts` array ordered? It should be alphabetical if not already

Reply:
yes it's sorted already